src-docs/src/views/provider/provider_example.js

```diff
@@ -121,7 +121,9 @@ export const ProviderExample = {
             <EuiCode>utility</EuiCode> properties on the{' '}
             <EuiCode>cache</EuiCode> prop to further define where specific
             styles should be inserted. See{' '}
-            <EuiLink href="#euiprovider-props">the props documentation</EuiLink>{' '}
+            <EuiLink href="#/utilities/provider#euiprovider-props">
```

Comment:
Ah thanks for catching this! The issue here is that this component needs to use React Router's `Link` component which automatically accounts for correct hashing:

```tsx
import { Link } from 'react-router-dom';
```

```suggestion
            <Link to="#euiprovider-props">
```